app/src/pages/SellingPage/Wizard/SearchForBookWindowCard.tsx

```diff
@@ -0,0 +1,54 @@
+import { Box, Paper, Stack, Typography, Button } from '@mui/material';
+import { Book } from 'cremona';
+import React from 'react';
+import EastOutlinedIcon from '@mui/icons-material/EastOutlined';
+import SearchBook from '../../../components/SearchBook';
+
+interface SearchForBookWindowCardProps {
+  setBook: Function;
+  handleNext: Function;
+  active: boolean;
+  canNext: boolean;
+}
+
+export default function SearchForBookWindowCard({
+  setBook,
+  handleNext,
+  active,
+  canNext,
+}: SearchForBookWindowCardProps) {
+  return (
+    <Box flexGrow={1}>
+      <Paper elevation={5}>
+        <Stack
+          bgcolor="white"
+          alignItems="center"
+          spacing={5}
+          padding={2}
+          paddingTop={5}
+          borderRadius={2}
+          height="335px"
+        >
+          <Typography textAlign="center" variant="h2">
+            Get started
```

Comment:
Might aswell start changing the text to swedish

Reply:
Everything should be in swedish now